docs/cody/capabilities/ignore-context.mdx

```diff
@@ -1,10 +1,151 @@
 
-## Enabling Unstable Features
+-   Cody context filters are currently supported only by Cody Web.
+-   Starting from May 27, support for Cody context filters will also be available on the Cody VS Code Extension and the Cody JetBrains Extension.
+-   A minimum required client version is enforced for these extensions: `>=1.20.0` for the VS Code extension and `>=6.0.0` for the JetBrains extension.
+-   If Cody context filters are defined in the site configuration, other Cody clients will not function as they do not support this feature.
+-   The feature is disabled by default until the required extensions versions are released. Administrators wishing to enable Cody context filters must activate the "cody-context-filters-enabled" feature flag. Be aware that enabling this feature may prevent VS Code and JetBrains clients that do not meet the version constraint from working.
+
+</Callout>
+
+Enterprise instances admins can define specific rules to determine which repositories Cody is allowed to utilize for context in its requests to third-party LLMs.
+
+To establish these rules, administrators should configure the `cody.contextFilters` field in the site configuration.
+
+The field structure is as follows:
+
+```json
+{
+	"cody.contextFilters": {
+		"include": [
+			{
+				"repoNamePattern": "^github\\.com\\/sourcegraph\\/.*"
+			}
+		],
+		"exclude": [
+			{
+				"repoNamePattern": ".*analytics.*"
+			}
+		]
+	}
+}
+```
+
+### Field validation
+
+-   The `cody.contextFilters` field is optional.
+-   If specified, it must include at least one of the following fields: `include` or `exclude`.
+-   Only the `include` and `exclude` properties are permitted; no additional properties are allowed.
+-   Each defined `include` or `exclude` field must contain at least one item.
+-   Every item within the `include` and `exclude` fields must specify a `repoNamePattern`.
+-   The `repoNamePattern` must conform to the regular expression syntax supported by [RE2](https://golang.org/s/re2syntax).
+
+### Interaction between include and exclude rules
+
+1.  **`cody.contextFilters` field is not defined**
+
+    By default, there are no restrictions on the repositories Cody can use for context in requests to third-party LLMs.
+    All repositories are included and none are excluded.
+
+2.  **Only `include` rules are specified**
+
+    Cody is restricted to using content from repositories whose names match any of the specified patterns in the include field.
+
+    Since no exclude rules are specified, no repositories are explicitly excluded.
+
+    Note: An include field might contain a catch-all pattern (e.g., a regexp matching any string). In this scenario,
+    Cody can access content from any repository, as a single match in the include rules is sufficient for inclusion.
+
+    Example:
+
+```json
+{
+	"cody.contextFilters": {
+		"include": [
+			{"repoNamePattern": "^github\\.com\\/sourcegraph\\/.+"},
+			{"repoNamePattern": ".*cody.*"}
+		]
```

Comment:
@taras-yemets I like this section, but it might be more powerful to use comments in the config file again to highlight what is happening. It feels like it will be easier to comprehend for me. At least that is my impression, we can ask others re: this too